app/services/custom_fields/create_service.rb

```diff
@@ -29,7 +29,7 @@
 module CustomFields
   class CreateService < ::BaseServices::Create
     def self.careful_new_custom_field(type)
-      if type.to_s =~ /.+CustomField\z/
+      if /.+CustomField\z/.match?(type.to_s)
```

Comment:
Wouldn't `type.to_s.end_with?('CustomField')` be better here?

Replying to myself: probably not, because `CustomField` does not match in the original case, but matches in my suggested change. So the behavior would be different.

You can ignore my comment :)